internal/models/payment_status.go

```diff
@@ -56,7 +56,9 @@ func (t PaymentStatus) String() string {
 	case PAYMENT_STATUS_DISPUTE_LOST:
 		return "DISPUTE_LOST"
 	case PAYMENT_STATUS_AMOUNT_ADJUSTMENT:
-		return "AMOUNT_ADJUSTMENT"
+		// There is a known typo here, but changing it would break old SDKs that expect this value.
+		return "AMOUNT_ADJUSTEMENT"
```

Comment:
You'll need to reverse the changes made to the open API spec for this to matter.